### PR TITLE
Small fixes and additions

### PR DESCRIPTION
**Pull request by Dotsially**
### Summary
Updates New Player prefab so it works after the shotgun charge merge.
Updates the game manager singleton to look like the other singletons.
Adds a Manager prefab so its easier to add to new scenes.
Organized folder structure.

### 1. Merging
- [x] Did you select the dev branch?
### 2. File Structure
- [x] Are all scripts placed within Assets/Scripts and organized into logical categories using PascalCase (e.g., Core, Player, Level)?
- [x] Are assets like materials, textures and sound files named using lower_snake_case (e.g., diamond_pickaxe.png)?
- [x] Did the changes avoid unnecessary scene modifications? If new changes were needed for testing, were they done in Scenes/Dev/<Specification> and then discarded?
### 3. Code
- [x] Do private fields start with an underscore and use camelCase (e.g., _privateVariable)?
- [x] Are static readonly variables named using UPPER_SNAKE_CASE?
- [x] Are enum types also in UPPER_SNAKE_CASE?
- [x] Are function and variable names clear, concise, and descriptive without using unnecessary abbreviations?
- [x] Are all public functions and fields documented with clear summaries where needed
- [x] Does the code maintain a strict hierarchy, ensuring objects do not fetch references themselves or their parents in the hierarchy?
- [x] Are references passed down through dependency injection as needed?Does each component only depend on its parent in the hierarchy?
- [x] Have unnecessary/testing Debug.Logs been removed?
- [x] Has commented code been removed?
### 4. Logging
- [ ] If possible, did you make a video and did you send it/a link to it in the feature-videos channel?

### CODE REVIEW

File: Project/Assets/Scripts/Managers/GameManager.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+public class GameManager : MonoBehaviour
+{
+    public bool DoDebugRendering;
+
+    public static GameManager Instance;
+
+    [HideInInspector]
+    public Camera MainCamera;
+
+    private void Awake()
```
Comment (ninyauwu) [inline Project/Assets/Scripts/Managers/GameManager.cs:16]: You made GameManager.cs a singleton? To what end?

**Review by ninyauwu — CHANGES_REQUESTED** [review]
You moved GameManager but git hasn't marked it as moved, this will lead to merge conflicts down the line. Please mark it as such.
Also, why exactly did you make GameManager a Unity singleton?

File: Project/Assets/Scripts/Managers/GameManager.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor.Experimental.GraphView;
```
Comment (IcyKoala) [inline Project/Assets/Scripts/Managers/GameManager.cs:3]: Pretty sure this GraphView is smthn that the editor added by itself for some reason, it also seems to interfere with building the game, so prolly remove this if it has no reason for existing.

Comment (Dotsially) [reply Project/Assets/Scripts/Managers/GameManager.cs]: Was already in the script. Removed it tho 👍

File: Project/Assets/Scripts/Managers/GameManager.cs (hunk 1 above)
Comment (Dotsially) [reply Project/Assets/Scripts/Managers/GameManager.cs]: Reverted it to old version.

**Review by ninyauwu — APPROVED** [review]
Ye, looks good

**Pull request merged**